### PR TITLE
feat(progress): make full/empty fill characters configurable

### PR DESCRIPTION
**Pull request by rwinkhart**
This is a simple change adding a function, WithCustomFillCharacters, to configure the characters the progress bar is made from. The function takes two runes as parameters for setting m.Full and m.Empty, respectively.

In other words, this adds the option to create a progress bar using any desired characters to represent the full and empty portions of the bar.

This is my first contribution to a larger public project, so please let me know if I went about creating this pull request correctly.

### CODE REVIEW

File: progress/progress.go [hunk 1]
```diff
@@ -79,6 +79,14 @@ func WithSolidFill(color string) Option {
 	}
 }
 
+// WithCustomFillCharacters sets the characters used to construct the full and empty components of the progress bar.
+func WithCustomFillCharacters(full rune, empty rune) Option {
```
Comment (meowgorithm) [inline progress/progress.go:83]: Mind changing this to simply `WithFillCharacters`?

**Review by meowgorithm — CHANGES_REQUESTED** [review]
Looks good! Would just rename that function and doc comment and we should be good to go.

File: progress/progress.go (hunk 1 above)
Comment (rwinkhart) [reply progress/progress.go]: Done! Thanks for taking a look at this!

**Review by meowgorithm — APPROVED** [review]

**Comment by meowgorithm** [comment]
Done and done. Thanks for the contribution!

**Pull request merged**